modules/mobile/docs/schemas/DiscoveryResponse.yml

```diff
@@ -0,0 +1,55 @@
+type: object
+required:
+  - data
+properties:
+  data:
+    type: object
+    required:
+      - id
+      - type
+      - attributes
+    properties:
+      id:
+        type: string
+        example: "1.0"
+        description: API Version number
+      type:
+        type: string
+        example: "discovery"
+      attributes:
+        type: object
+        required:
+          - endpoints
```

Comment:
can we add apiRoot here as well, since we are returning different values for dev/staging and prod?

Reply:
Added